client/stylesheets/_toCat.scss

```diff
@@ -2157,6 +2158,7 @@ i:hover {
     overflow: hidden;  
     font-size: 1.2rem !important; 
     float: right !important;
+    font-size: 1rem !important;
```

Comment:
Property `font-size` already defined on line 2159<br>!important should not be used<br>Line should be indented 2 spaces, but was indented 4 spaces